src/CentreonRemote/Application/Webservice/CentreonTaskService.php

```diff
@@ -179,14 +189,6 @@ public function postAddImportTaskWithParent()
         try {
             if (!$parent_id) {
                 throw new Exception('Missing parent_id parameter');
-            } elseif (!$params) {
-                throw new Exception('Missing params parameter');
-            } elseif (!array_key_exists('pollers', $params)) {
-                throw new Exception('Missing pollers list in params');
-            } elseif (!array_key_exists('pollers', $params)) {
-                throw new Exception('Missing pollers list in params');
-            } elseif (!array_key_exists('server', $params)) {
-                throw new Exception('Missing server ID in params');
```

Comment:
Do we have validation of $params in the `addTask` method?

Reply:
no, it just needs to be an array, can be an empty one